templates/default-browser-agent/default-browser/default-browser.1.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "build_channel": {
+      "type": "string"
+    },
+    "build_version": {
+      "type": "string"
+    },
+    "os_version": {
+      "description": "Windows version numnber in major.minor.build.UBR format (UBR is optional, only available on Win10)",
+      "type": "string"
+    },
+    "os_locale": {
+      "description": "The current user's OS locale setting",
+      "type": "string"
+    },
+    "default_browser": {
+      "description": "The current default browser",
+      "type": "string",
+      "enum": [ "firefox", "chrome", "edge", "edge-chrome", "ie", "opera", "brave" ]
+    },
+    "previous_default_browser": {
+      "description": "What the default browser was before it was changed to the current one",
+      "type": "string",
+      "enum": [ "firefox", "chrome", "edge", "edge-chrome", "ie", "opera", "brave" ]
```

Comment:
unless these values effectively aren't subject to change, I would be inclined to not enforce this as part of schema validation.

Reply:
Okay, I'll take out the enums. Thanks!